projects/demo/src/modules/app/pages.ts

```diff
@@ -1326,4 +1324,4 @@ export const pages: TuiDocPages = [
             },
         ],
     },
-];
```

Comment:
So we can make a mistake in this config and the error will not be highlighted, why is this better?

Reply:
for proprietary repo
<img width="980" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/de3e2722-104c-4b8e-ac84-e1e99c6b0a4b">